server/testsupport/prisonerSearchSummaryTestDataBuilder.ts

```diff
@@ -0,0 +1,32 @@
+import moment from 'moment'
+import type { PrisonerSearchSummary } from 'viewModels'
+
```

Comment:
This is quite a neat pattern for our test data builders, and makes them a bit more like what we do in kotlin 👍 
What we do is pass an optional map of "options" into the test data builder function, where each "option" is a named field. All of the options are optional. If a given option value is not provided then it gets a default value (though to be fair the default values themselves are not as clear and obvious as they are in our beloved kotlin 🤷‍♂️ )

Anyway you can do this:
```
const defaultPrisoner = aValidPrisonerSearchSummary() // A1234BC, BXI, Jimmy Lightfingers etc)
const jimmyInMoorland = aValidPrisonerSearchSummary({ prisonId: 'MDI' }) // A1234BC, MDI, Jimmy Lightfingers etc)
const aDifferentJimmyInADifferentPrisoner = aValidPrisonerSearchSummary({ prisonNumber: 'T4385GH', prisonId: 'MDI' }) // T4385GH, MDI, Jimmy Lightfingers etc)
const anEntirelyDifferentPrisoner = aValidPrisonerSearchSummary({ prisonNumber: 'Y8310JK', prisonId: 'RDG', lastName: 'Mahone', firstName: 'James' }) // Y8310JK, RDG, James Mahone etc)
```

You only need to specify the options that you want to change from the defaults, and because they are named arguments you can specify them in any order .... just like a kotlin function 👍 😁